tasks/main.yml

```diff
@@ -27,4 +27,14 @@
   action: apt pkg="{{ item }}" state=present
   with_items:
     - unattended-upgrades
-  when: base_is_server
\ No newline at end of file
+  when: base_is_server
+
+- name: Set timezone variables
+  copy: content="{{ base_timezone }}"
+        dest=/etc/timezone
+        owner=root
+        group=root
+        mode=0644
+        backup=yes
+  notify:
+    - update timezone
```

Comment:
Minor: Missing NL at the end of file.
Don't forget to make it upercase here also.